.github/workflows/test-application.yaml

```diff
@@ -161,3 +161,81 @@ jobs:
             - name: Run tests
               run: time ./bin/runtests -i -a ${{ matrix.test-flags }}
               env: ${{ matrix.env }}
+
+    test-windows:
+        name: "PHP ${{ matrix.php-version }} on Windows (${{ matrix.phpcr-transport }}, ${{ matrix.dependency-versions }})"
+        runs-on: windows-latest
+
+        env:
+            JACKRABBIT_VERSION: 2.20.2
+            SYMFONY_DEPRECATIONS_HELPER: weak
+            SYMFONY__PHPCR__TRANSPORT: jackrabbit
+            SYMFONY__DATABASE__DRIVER: pdo_mysql
+            SYMFONY__DATABASE__VERSION: 5.7
+            SYMFONY__DATABASE__HOST: 127.0.0.1
+            SYMFONY__DATABASE__PORT: 3306
+            SYMFONY__DATABASE__USER: root
+            SYMFONY__DATABASE__PASSWORD: root
+
+        strategy:
+            fail-fast: false
+            matrix:
+                include:
+                    - php-version: '7.2'
+                      phpcr-transport: jackrabbit
+                      dependency-versions: 'highest'
+                      test-flags: '--jackrabbit-restart' # restart jackrabbit after each suite see: https://github.com/sulu-io/sulu/issues/2137
```

Comment:
does this really work on windows? 😮

Reply:
I'm not sure, maybe this isn't even needed ... Just had a look at the old `appveyor.yml`, this option wasn't passed there

Reply:
i would suspect that at least the following command does not work on windows 🙈 
https://github.com/sulu/sulu/blob/1.6/bin/runtests#L292

so if it works without the flag, i think i would remove it

Reply:
```suggestion
```